### PR TITLE
Fix Xcode 13 build (alternate PR to kick CI)

### PR DESCRIPTION
**Pull request by mthole**
This is just a copy of https://github.com/yonaskolb/XcodeGen/pull/1129, but from my fork so I can attempt to kick CI a few times and see if I can figure out what's amiss.

### CODE REVIEW

**Comment by mthole** [comment]
Ah well, foo, that didn't work. @yonaskolb, could you approve the workflow run?

**Comment by mthole** [comment]
This is now green on my fork: https://github.com/mthole/XcodeGen/actions/runs/1267184165

image

**Comment by freddi-kit** [comment]
CI is running after Collaborator approval, I approved the original repo to check it. Would you mind closing this PR?

**Comment by freddi-kit** [comment]
Sorry I found you have a workaround for Xcode13, let me check it

**Comment by freddi-kit** [comment]
i just approved it

**Comment by mthole** [comment]
🤞

File: CHANGELOG.md
Comment (freddi-kit) [reply CHANGELOG.md:15]: And please write your name with @raptorxcz  :)

**Comment by freddi-kit** [comment]
All Green 👍

**Review by freddi-kit — APPROVED** [review]
Looks good to me, Thank you for first contribution @mthole and nice fixing @raptorxcz !

**Comment by freddi-kit** [comment]
@yonaskolb I think we can merge it

File: CHANGELOG.md
Comment (freddi-kit) [reply CHANGELOG.md]: When writing about changes, the changes before releasing should be placed in the New Version column. 
Before the releasing, maintainer changes the New Version with new one (i expect 2.26.0 in this case)

FYI: https://github.com/yonaskolb/XcodeGen/commit/c3d936c0c7d42f906d35dde41a2ee887f680b59a

Comment (freddi-kit) [reply CHANGELOG.md]: ah sorry. its `Next Version`

File: .github/workflows/ci.yml
Comment (freddi-kit) [inline .github/workflows/ci.yml:11]: Do we still need 12.5.1? @yonaskolb  I think it is only takes long time to build

Comment (mthole) [reply .github/workflows/ci.yml:11]: I think it's wise to leave both 12.5.x and 13.0, at least for a few weeks until Xcode 13 adoption is mainstream. It's a nice guard against accidentally regressing one of the major Xcode builds out there. And the builds run in parallel, so it doesn't actually take longer AFAICT.

I'd suggest revisiting and removing 12.x in a couple months.

File: CHANGELOG.md
Comment (mthole) [reply CHANGELOG.md]: Roger that, I'll move the line up!

Comment (freddi-kit) [reply CHANGELOG.md]: Thank you so much! 😄

File: .github/workflows/ci.yml
Comment (yonaskolb) [reply .github/workflows/ci.yml:11]: Agree, let's keep till there's at least a 13.1

File: CHANGELOG.md
Comment (yonaskolb) [reply CHANGELOG.md]: I will actually go under 2.25.0 as I pre-emptively added that heading yesterday. Will fix in master

**Review by yonaskolb — APPROVED** [review]
Thanks all! 🙏

**Pull request merged**
